README.md

```diff
@@ -16,11 +16,13 @@ See the [wiki](https://github.com/richardwilly98/elasticsearch-river-mongodb/wik
 
 In order to install the plugin, simply run: ```bin/plugin --install com.github.richardwilly98.elasticsearch/elasticsearch-river-mongodb/2.0.9```
 
+[El Past Releases](https://www.elastic.co/downloads/past-releases)
```

Comment:
Yes, I know. I wasn't suggesting changing the link url. Only the link text, which is confusing